db/migrate/20210419200234_add_hearing_day_slot_columns.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddHearingDaySlotColumns < Caseflow::Migration
+  def change
+    add_column :hearing_days, :number_of_slots, :integer, comment: "The number of slots possible for this day"
```

Comment:
```suggestion
    add_column :hearing_days, :number_of_slots, :integer, comment: "The number of time slots possible for this day"
```
Also consider replacing "possible" with "available" if appropriate.

Reply:
Noting that I'm sticking with 'possible' over 'available' because in other work the front-end makes a distinction between 'possible' and 'available' timeslots.
- 'possible' timeslots is the list of every time slot the day could have based on the length of slots and number of slots.
- 'available' timeslots is a subset of that list representing the timeslots that show up in the UI as available for a user to schedule to.